js/layout-javascript/agenda-code.js

```diff
@@ -756,6 +756,11 @@ DynamicList.prototype.attachObservers = function() {
         Fliplet.UI.Actions(options);
       });
     })
+    .on('click', '.file-item', function(file) {
+      var url = $(file.currentTarget).find('input[type=hidden]').val();
```

Comment:
```suggestion
      var url = $(file.target).find('input[type=hidden]').val();
```

Reply:
@tonytlwu in our case it would be better to leabe with current traget whitch always will be `li.file-item`. If we use `target` then click may go on child items of the .file-item and our next operation `$(file.target).find('input[type=hidden]').val();` will endup undefined whitch will be leading to the error in this function `Fliplet.Navigate.file(url);`

Reply:
I see. Let's use `event.target` instead of `file.target` then. The parameter to the event listener should always be `event`. Using `file` could be misleading.

Reply:
Not addressed